src/components/Comments.jsx

```diff
@@ -0,0 +1,62 @@
+import CommentCard from "./CommentCard"
+import { useState, useEffect } from "react"
+import { useParams } from "react-router"
+import { fetchCommentsByArticleId, postCommentOnArticle } from "./api"
+
+function Comments({article}) {
+    const {article_id} = useParams()
+    const [articleComments, setArticleComments] = useState([])
+    const [commentText, setCommentText] = useState("")
+    const [newComment, setNewComment] = useState({})
+    const [isCommentButtonDisabled, setCommentButtonDisabled] = useState(false)
+
+    useEffect(() => {
+        fetchCommentsByArticleId(article_id).then((res) => {
+            setArticleComments(res.data.comments)
+        })
+    }, [article_id])
+
+    function handleChange(event) {
+        setCommentText(event.target.value)
+    }
+
+    function disableCommentButton() {
+        setCommentButtonDisabled(true)
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault()
+        postCommentOnArticle(article_id, commentText).then(() => {
+            setCommentText("")
+            fetchCommentsByArticleId(article_id).then((res) => {
+                setArticleComments(res.data.comments)
+            })
+            disableCommentButton()
+        })
+    }
+
+    return (
+        <section>
+            <form className="comment-form" onSubmit={handleSubmit}>
+                <label className="comment-label">comment:</label>
```

Comment:
you should use the property htmlFor when using labels and link them to the a corresponding input using the id. This is important for accessibility